scripts/validation/service.schema.js

```diff
@@ -116,6 +116,10 @@ const schema = {
               }
             }
           ]
+        },
+        executeClientScripts: {
+            type: 'boolean',
+            description: 'Optional boolean to use a puppeteer based fetcher, capable of gathering terms from websites using scripts to render content',
```

Comment:
Align description with the documentation.

```suggestion
            description: 'Execute client-side JavaScript loaded by the document before accessing the content, in case the DOM modifications are needed to access the content.',
```